client/lib/abtest/active-tests.js

```diff
@@ -125,5 +125,14 @@ module.exports = {
 		defaultVariation: 'showPlansAfterAuth',
 		allowExistingUsers: true
 	},
-};
 
+	presaleChatButton: {
+		datestamp: '20161129',
+		variations: {
+			showChatButton: 80,
+			original: 20
```

Comment:
@dllh I originally had `hideChatButton` but saw the usage of [`original`](https://github.com/Automattic/wp-calypso/blob/master/client/lib/abtest/active-tests.js#L85) in a few places and decided to go with the flow.